packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -878,7 +878,7 @@ class ClampingScrollPhysics extends ScrollPhysics {
       // Hit top edge.
       return value - position.minScrollExtent;
     }
-    if (position.pixels < position.maxScrollExtent && position.maxScrollExtent < value) {
+    if (position.pixels < position.maxScrollExtent && value - position.maxScrollExtent > precisionErrorTolerance) {
```

Comment:
Another thing that I'm wondering, why is overflowing by such a small value of `1.1368683772161603e-13` gives us the stetch of `1.440112717630772` and the stretch does not recede?

There might be another deeper problem with the `StretchingOverscrollIndicator`

@Piinks what do you think?